doc/release/RELEASE-NOTES.md

```diff
@@ -4,6 +4,36 @@ All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html). The format is based on the `RELEASE-NOTES-TEMPLATE.md` file.
 
 
+## Release 1.17.0
+
+## Introduction
+* Product name: Open Supply Hub
+* Release date: *Provide release date*
+
+### Database changes
+#### Migrations:
+* *Describe migrations here.*
+
+#### Scheme changes
+* *Describe scheme changes here.*
+
+### Code/API changes
+* *Describe code/API changes here.*
+
+### Architecture/Environment changes
+* *Describe architecture/environment changes here.*
+
+### Bugfix
+* *Describe bugfix here.*
+
+### What's new
+* [OSDEV-1083](https://opensupplyhub.atlassian.net/browse/OSDEV-1083) - Enable Password Visibility on Chrome Browser.
```

Comment:
Are these changes applicable only to the Chrome browser?

In my opinion, it would be better to remove "Enable Password Visibility on Chrome Browser" and keep only the explanation.

Also, you forgot to add that this feature was implemented on the "Reset Password" page as well.